buildflow/core/app/runtime/actors/pipeline_pattern/pull_process_push.py

```diff
@@ -250,14 +282,16 @@ async def drain(self):
         return True
 
     async def snapshot(self):
+        throughput = self.num_events_processed.calculate_rate()
         snapshot = PullProcessPushSnapshot(
             status=self._status,
             timestamp_millis=utils.timestamp_millis(),
-            events_processed_per_sec=self.num_events_processed.calculate_rate(),  # noqa: E501
+            events_processed_per_sec=throughput,  # noqa: E501
```

Comment:
can we keep this as self.num_events_processed.calculate_rate() for consistency?

Reply:
could add a `# represents throughput` comment or something

Reply:
oops, yea I had pulled this out for debugging and forgot to put it back